### PR TITLE
Adding jobs example.

### PR DESCRIPTION
**Pull request by meghamsh4**
Tested with job_id : Search results returns fine 
On job_description : No results or cassandra error

### CODE REVIEW

File: sodra-backend/scripts/examples/jobs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	salary decimal,
+	location text,
+	job_description text,
+	qualifications text,
+	skills text,
+	posting_date timestamp 
+);
+
+COPY sodra.jobs(job_id,agency,job_title,salary,location,job_description,qualifications,skills,
+posting_date) from 'Jobs.csv' with header = true;
```
Comment (redragons) [inline sodra-backend/scripts/examples/jobs]: batch size missing

File: sodra-backend/scripts/examples/jobs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+CREATE CUSTOM INDEX IF NOT EXITS user_idx ON sodra.user(job_description)
```
Comment (redragons) [inline sodra-backend/scripts/examples/jobs]: spell mistake for EXITS. Should be EXISTS

Comment (redragons) [inline sodra-backend/scripts/examples/jobs]: table should be ```sodra.jobs```

Comment (redragons) [inline sodra-backend/scripts/examples/jobs]: nit: also change the index name from users_idx to something like jobs_idx

**Comment by meghamsh4** [comment]
Fixed jobs script

**Pull request merged**